rdflib/plugins/sparql/operators.py

```diff
@@ -617,17 +617,21 @@ def Function(e, ctx):
     Custom functions and casts
     """
     pair =_CUSTOM_FUNCTIONS.get(e.iri)
-    if pair is not None:
-        func, raw = pair
-        if raw:
-            return func(e, ctx)
-        else:
-            try:
-                return func(*e.expr)
-            except TypeError as ex:
-                raise SPARQLError(*ex.args)
+    if pair is None:
+        # no such function is registered
+        raise SPARQLError('Unknown function %r"%e.iri')
```

Comment:
i think what you actually want is `raise SPARQLError('Unknown function %r' % e.iri)`